app/connectors/des/DesConnector.scala

```diff
@@ -42,14 +41,13 @@ class DesConnector @Inject() (servicesConfig: ServicesConfig, httpClient: HttpCl
   private val desHeaderCarrier: HeaderCarrier = HeaderCarrier(authorization = Some(Authorization(s"Bearer $authorisationToken")))
     .withExtraHeaders("Environment" -> serviceEnvironment)
 
-  //may not need this one
-  def getObligations(vrn: String, fromDate: LocalDate, toDate: LocalDate): Future[VatObligations] = {
-    Logger.debug(s"Calling des api 1330 for vrn ${vrn}, fromDate ${fromDate.toString}, toDate ${toDate.toString}")
+  def getObligations(vrn: String): Future[Option[VatObligations]] = {
+    Logger.debug(s"Calling des api 1330 for vrn ${vrn}")
     implicit val hc: HeaderCarrier = desHeaderCarrier
     val fullDate = DateTimeFormatter.ofPattern("yyyy-MM-dd")
-    val getObligationsURL: String = s"$serviceURL$obligationsUrl/${vrn}/VATC?from=${fromDate.format(fullDate)}&to=${toDate.format(fullDate)}&status=F"
+    val getObligationsURL: String = s"$serviceURL$obligationsUrl/${vrn}/VATC?status=O"
```

Comment:
ooops, I probably missed that out. This is frontend microservice thus is in public zone. In order to access DES API we need to call it from protected zone, which means that this connector will have to go to some backend microservice. 
Let's address it in another PR.